### PR TITLE
PJ_SIM_SUNBNB_2023-25 [FE] User Profile

### PR DESCRIPTION
**Pull request by ThereseRoca**
# Issue Link
[PJ_SIM_SUNBNB_2023-25 [FE] User Profile](https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-25)

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1691147422055179

# Definition of Done
Create a markup for the user profiles with the components:
- [x] Full Name
- [x] User Image
- [x] Phone Number
- [x] Description
- [ ] List of Owned Listings (if Host)
- [x] List of Made Reviews (if Client)

# To Do
- Show owned listings of the user if he is a host when the listing item component is merged

# Expected Output
Host
image

Client
image

# Screenshots

Client View
image

Host View
image

### CODE REVIEW

File: fe/src/App.vue [hunk 1]
```diff
@@ -15,4 +15,6 @@ export default {
     NavBar,
   },
 };
+
```
Comment (roseaugusto) [inline fe/src/App.vue]: is this space necessary?

File: fe/src/components/Navbar/NavbarProfile.vue [hunk 2]
```diff
@@ -54,6 +54,9 @@ export default {
     },
     viewProfile() {
       // Implement action to view profile
+      this.$router.push('/ClientViewProfile')
+
```
Comment (roseaugusto) [inline fe/src/components/Navbar/NavbarProfile.vue]: remove extra spaces

File: fe/src/pages/ClientViewProfile.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+});
+
+//To do
+//Retrieve all the reviews made by the user then show it using the review component
+
+
+</script>
+
+<style scoped>
+
```
Comment (roseaugusto) [inline fe/src/pages/ClientViewProfile.vue]: remove unnecessary spaces here same with the bottom.

**Review by roseaugusto — CHANGES_REQUESTED** [review]

File: fe/src/App.vue (hunk 1 above)
Comment (ThereseRoca) [reply fe/src/App.vue]: I removed the extra space already po ma'am.

File: fe/src/components/Navbar/NavbarProfile.vue (hunk 2 above)
Comment (ThereseRoca) [reply fe/src/components/Navbar/NavbarProfile.vue]: I removed the extra space already po ma'am.

File: fe/src/pages/ClientViewProfile.vue (hunk 3 above)
Comment (ThereseRoca) [reply fe/src/pages/ClientViewProfile.vue]: I removed the extra space already po ma'am.

File: fe/src/components/Tab/HostTab.vue [hunk 4]
```diff
@@ -1,47 +1,53 @@
...
 <script setup>
-  let isActive = true
-
-  function changeActive(option) {
+let isActive = true;
 
-    switch(option){
-      case "list": isActive = true; 
+function changeActive(option) {
+  switch (option) {
```
Comment (roseaugusto) [inline fe/src/components/Tab/HostTab.vue]: is parameter `option` only has two type? if yes, please change to boolean and you can change name to `isListing` or `isReservation`

Comment (andreenriquez11) [reply fe/src/components/Tab/HostTab.vue]: Changed it already maam

**Review by roseaugusto — APPROVED** [review]

**Pull request merged**